src/test.js

```diff
@@ -334,7 +339,7 @@ Test.prototype = {
 			testId: this.testId,
 
 			// Source of Test
-			source: this.stack
+			source: this.getStack()
```

Comment:
Hmm, do we need the source of the test here? Seems like this particular usage of the stack is not conditional and would ultimately force all tests to generate the stack even if it were unused.

Thoughts on:

```js
return runLoggingCallbacks('testDone', {
  //...snip...
  get source() { return this.getStack(); }
});
```

That way if the source _wasn't_ used by the logging callbacks, the stack would never have to be reified...